src/SB/Game/zMain.cpp

```diff
@@ -608,6 +642,92 @@ void zMainLoop()
     U32 preinit_bubble_matfx[6];
     U32 preinit_shiny_models[8];
     U32 preinit_ADC_models[10];
+
+    zMainShowProgressBar();
+    xMemPushBase();
+    time = iTimeGet();
+    xUtil_idtag2string(0x464f4e54, 0);
+    iTimeDiffSec(time);
+    xSTPreLoadScene(0x464f4e54, NULL, 0x1);
+    time = iTimeGet();
+    xUtil_idtag2string(0x464f4e54, 0);
+    iTimeDiffSec(time);
+    xSTQueueSceneAssets(0x464f4e54, 1);
+    time = iTimeGet();
+    xUtil_idtag2string(0x464f4e54, 0);
+    iTimeDiffSec(time);
+
+    do
+    {
+        xSTLoadStep(time);
+    } while (time < 1.0f);
+
+    xSTDisconnect(0x464f4e54, 1);
+    time = iTimeGet();
+    xUtil_idtag2string(0x464f4e54, 0);
+    iTimeDiffSec(time);
+    zMainShowProgressBar();
+    xSTPreLoadScene(0x504c4154, 0, 1);
+    xSTQueueSceneAssets(0x504c4154, 1);
+
+    do
+    {
+        xSTLoadStep(time);
+    } while (time < 1.0f);
+
+    xSTDisconnect(0x504c4154, 1);
+    zMainShowProgressBar();
+    iTimeGet();
+    xShadowSimple_Init();
+    globals.pickupTable = (zAssetPickupTable*)xSTFindAssetByType(0x5049434b, 0, 0);
+    // globals.pickupTable = zPickupTableInit();
+    // zPickupTableInit hasnt been implemented yet
+    xMemPushBase();
+    time = iTimeGet();
+    xUtil_idtag2string(0x4d4e5534, 0);
+    iTimeDiffSec(time);
+    xSTPreLoadScene(0x4d4e5534, 0, 1);
+    time = iTimeGet();
+    xUtil_idtag2string(0x4d4e5534, 0);
+    iTimeDiffSec(time);
+    xSTQueueSceneAssets(0x4d4e5534, 1);
+    time = iTimeGet();
+    xUtil_idtag2string(0x4d4e5534, 0);
+    iTimeDiffSec(time);
+    do
+    {
+        xSTLoadStep(time);
+    } while (time < 1.0f);
+    xSTDisconnect(0x4d4e5534, 1);
+    zMainShowProgressBar();
+    time = iTimeGet();
+    xUtil_idtag2string(0x4d4e5534, 0);
+    iTimeDiffSec(time);
+    xMemPushBase();
+    time = iTimeGet();
+    xUtil_idtag2string(0x4d4e5535, 0);
+    iTimeDiffSec(time);
+    xSTPreLoadScene(0x4d4e5535, 0, 1);
+    time = iTimeGet();
+    xUtil_idtag2string(0x4d4e5535, 0);
+    iTimeDiffSec(time);
+    xSTQueueSceneAssets(0x4d4e5535, 1);
+    time = iTimeGet();
+    xUtil_idtag2string(0x4d4e5535, 0);
+    iTimeDiffSec(time);
+    do
+    {
+        xSTLoadStep(time);
+    } while (time < 1.0f);
+    xSTDisconnect(0x4d4e5535, 1);
+    zMainShowProgressBar();
+    time = iTimeGet();
+    xUtil_idtag2string(0x4d4e5535, 0);
+    iTimeDiffSec(time);
+    xModelInit();
+    xModelPoolInit(0x20, 0x40);
+    xModelPoolInit(0x28, 8);
+    xModelPoolInit(0x38, 1);
```

Comment:
Make sure that any integer that can be better expressed by a char literal (e.g 'PLAT', 'FONT') is substituted with that char literal here.